src/Components/TimeSignature/timeSignature.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+
+import styled from "styled-components";
+
+const TimeSigStyle = styled.div`
+  display: flex;
+  flex-direction: row;
+  input {
+    width: 25px;
+    height: 25px;
+    font-size: 16px;
+    text-align: end;
+    margin: 10px;
+  }
+`;
+
+const TimeSignature = ({
+  beatDivision,
+  beatsMeasure,
+  setBeatDivision,
+  setBeatsMeasure
+}) => {
+  return (
+    <>
+      <TimeSigStyle>
+        <p>Time Signature: </p>
+        <input
```

Comment:
Instead of allowing any kind of time signature let's restrict the user to the 4 most common time signatures. This includes 3/4, 4/4, 6/8, and 2/4. We just recently got a design for how this will look: https://www.figma.com/file/rXOaRB4nVWc2C5Lt5mKDqX/Drum-Root?node-id=52%3A25. If you have more questions on the design please contact @linconkusunoki :)

- [ ] Switch inputs to buttons